src/README.md

```diff
@@ -760,11 +760,19 @@ a `void* hint` argument.
 Inside these cleanup hooks, new asynchronous operations _may_ be started on the
 event loop, although ideally that is avoided as much as possible.
 
-Every [`BaseObject`][] has its own cleanup hook that deletes it. For
-[`ReqWrap`][] and [`HandleWrap`][] instances, cleanup of the associated libuv
-objects is performed automatically, i.e. handles are closed and requests
+For every [`ReqWrap`][] and [`HandleWrap`][] instance, cleanup of the associated
+libuv objects is performed automatically, i.e. handles are closed and requests
 are cancelled if possible.
```

Comment:
```suggestion
libuv objects is performed automatically, (i.e. handles are closed and requests
are cancelled, if possible).
```

Reply:
Thanks for the suggestion but I don't think it is necessary to add parentheses here.